src/renderer/drawers/NFTViewerDrawer/index.js

```diff
@@ -0,0 +1,236 @@
+  z-index: 1;
+`;
+
+const NFTActions = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin: 12px 0px;
+  justify-content: center;
+`;
+
+const Separator = styled.div`
+  width: 100%;
+  height: 1px;
+  background-color: ${({ theme }) => theme.colors.palette.text.shade10};
+  margin: 24px 0px;
+`;
+
+const NFTAttributes = styled.div`
+  margin: 24px 0px;
+  display: flex;
+  flex-direction: column;
+`;
+
+function NFTAttribute({
+  title,
+  value,
+  skeleton,
+  separatorBottom,
+  separatorTop,
+}: {
+  title: string,
+  value: string,
+  skeleton?: boolean,
+  separatorBottom?: boolean,
+  separatorTop?: boolean,
+}) {
+  if (!skeleton && !value) return null;
+
+  return (
+    <React.Fragment>
+      {separatorTop ? <Separator /> : null}
+      <Text
+        mb={1}
+        lineHeight="15.73px"
+        fontSize={4}
+        color="palette.text.shade60"
+        ff="Inter|Regular"
+      >
+        {title}
+      </Text>
+      <Skeleton show={skeleton} width={120} minHeight={24} barHeight={10}>
+        <Text lineHeight="15.73px" fontSize={4} color="palette.text.shade100" ff="Inter|SemiBold">
+          <Pre>{value}</Pre>
+        </Text>
+      </Skeleton>
+      {separatorBottom ? <Separator /> : null}
+    </React.Fragment>
+  );
+}
+
+type NFTViewerDrawerProps = {
+  nftId: string,
+  isOpen: boolean,
+  height?: number,
+  onRequestClose: () => void,
+};
+
+export function NFTViewerDrawer({ nftId, isOpen, onRequestClose, height }: NFTViewerDrawerProps) {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const nft = useSelector(state => getNFTById(state, { nftId }));
+  const { status, metadata } = useNftMetadata(nft.collection.contract, nft.tokenId);
+  const show = useMemo(() => status !== "loaded", [status]);
+  const name = centerEllipsis(metadata?.nftName || nft.tokenId, 26);
+
+  // const onNFTSend = useCallback(() => {
+  //   dispatch(openModal("MODAL_SEND", { isNFTSend: true, nftId }));
+  // }, [dispatch, nftId]);
+
+  return (
+    <Box height={height}>
+      <NFTViewerDrawerContainer>
+        <NFTViewerDrawerContent>
+          <StickyWrapper top={0} pb={3} pt="24px">
+            <Text
+              ff="Inter|SemiBold"
+              fontSize={5}
+              lineHeight="18px"
+              color="palette.text.shade50"
+              uppercase
+              pb={2}
+            >
+              <Skeleton show={show} width={100} barHeight={10} minHeight={24}>
+                {metadata?.tokenName}
+              </Skeleton>
+            </Text>
+            <Text
+              ff="Inter|SemiBold"
+              fontSize={7}
+              lineHeight="29px"
+              color="palette.text.shade100"
+              uppercase
+              pb={5}
+            >
+              {name}
+            </Text>
+          </StickyWrapper>
+          <Skeleton show={show} width={393} minHeight={393}>
+            <Image nft={metadata} full />
+          </Skeleton>
+          <NFTActions>
```

Comment:
remove this if unused

Reply:
It's just temporary, we're just hiding the send buttons because LLC & LLM isn't ready yet and we want to release the display only. Will be removed for next release 👌